prepare

```diff
@@ -106,4 +106,16 @@ fi
 
 # Install gitlab-runner
 $SSH "$(sshUser)@${VM_IP}" sudo dnf clean all
-$SSH "$(sshUser)@${VM_IP}" sudo dnf install -y "https://gitlab-runner-downloads.s3.amazonaws.com/latest/rpm/gitlab-runner_$(runnerArch).rpm"
+
+# retry install because of a race condition in dnf/subscription-manager
```

Comment:
Can we have a retry limit? If something goes wrong, this will result into having dead machines that run needlessly for many hours.